web_src/js/utils.js

```diff
@@ -58,3 +58,10 @@ export function parseIssueHref(href) {
   const [_, owner, repo, type, index] = /([^/]+)\/([^/]+)\/(issues|pulls)\/([0-9]+)/.exec(path) || [];
   return {owner, repo, type, index};
 }
+
+// pretty-print a number using locale-specific separators, e.g. 1200 -> 1,200
+export function prettyNumber(num, locale = 'en-US') {
+  if (typeof num !== 'number') return '';
```

Comment:
1. `locale = 'en-US'` seems not a good idea, it's better to use browser default.
2. `typeof num !== 'number'`, maybe we had better support type-casting here, to support string value `'123'`

Reply:
1. The default is never going to be hit because `locale` is passed based on HTML attribute. It seemed more efficient to only read the HTML attribute once instead of every call to this function.